app/assets/javascripts/covid19/src/webapp/components/structures-table/DetailsDialog.tsx

```diff
@@ -34,36 +34,33 @@ export const DetailsDialog: React.FC<DetailsDialogProps> = React.memo(props => {
                 </ImgContainer>
 
                 <div>
-                    <ModifiedAccordion expandedAccordion={expandedAccordion} field="entities">
-                        {row.entities.length > 0 ? (
-                            <EntityCell data={data} row={row} moreDetails={false}></EntityCell>
-                        ) : (
-                            <NoCellsFound name="entities" />
-                        )}
+                    <ModifiedAccordion
+                        expanded={expandedAccordion}
+                        field="entities"
+                        isEmpty={_.isEmpty(row.entities)}
+                    >
+                        <EntityCell data={data} row={row} moreDetails={false}></EntityCell>
```

Comment:
[do not change] [note] On another level of abstraction, you'd pass `component={EntityCell}` as prop. This is very compact but fails when you need to tweak things of this inner component.

Reply:
Oh, I see, thanks for the advice